roles/activemq/molecule/default/molecule.yml

```diff
@@ -1,45 +1,20 @@
 ---
-scenario:
-  test_sequence:
-    - lint
-    - cleanup
-    - destroy
-    - syntax
-    - create
-    - prepare
-    - converge
-    - idempotence
-    - side_effect
-    - verify
-    - cleanup
-    - destroy
 dependency:
   name: galaxy
 driver:
   name: docker
 platforms:
   - name: instance
-    image: centos:7
+    image: ${MOLECULE_ROLE_IMAGE:-centos:7}
+    dockerfile: ../../../molecule/Dockerfile.j2
+    command: "/lib/systemd/systemd"
     privileged: true
     tmpfs:
       - /run
       - /run/lock
       - /tmp
     volume_mounts:
       - "/sys/fs/cgroup:/sys/fs/cgroup:ro"
-    command: /sbin/init
-    exposed_ports:
-      - 80/tcp
-      - 443/tcp
-      - 5432/tcp
-      - 8161/tcp
-      - 8080/tcp
-      - 8082/tcp
-      - 8083/tcp
-      - 8090/tcp
-      - 8099/tcp
-      - 8983/tcp
-      - 9090/tcp
     published_ports:
```

Comment:
I don't understand why we need all these ports published for activemq tests. Actually the activemq port itself is not even part of the list, and as we use the docker exec driver to test on localhost... I suspect neither of theese ports are really needed. Worth checking what's really needed here